### PR TITLE
feat: add lit-html

### PR DESCRIPTION
**Pull request by digitalsadhu**
Adds Lit HTML to the monorepo.

Lit Element is 16kb minified and Lit HTML is 7kb. 

I considered trying to optimise where Lit Element imports Lit HTML via Eik so that Lit Element would reduce in size to (I assume) 9kbish but I'm not convinced its worth it for 2 reasons:
* We would always need to preload lit-html any time we use lit-element or suffer the esm waterfall load effect
* We would need to depend on the exact version of lit-html that is declared in the lit-element package's package.json dependencies and this version might not be the same version that is our latest aliased version of lit-html in use which:
  a. would defeat the optimisation
  b. would make things more complicated for publishing.

In other words, the value didn't seem there for 7kb.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -25,6 +25,10 @@
         {
           "groupName": "React and TrackJS packages",
           "packageNames": ["@esm-bundle/react", "@esm-bundle/react-dom", "trackjs"]
+        },
+        {
+          "groupName": "Lit packages",
+          "packageNames": ["lit-html", "lit-element"]
```
Comment (digitalsadhu) [inline renovate.json:31]: Ensures that both Lit packages get updated together

**Review by trygve-lie — APPROVED** [review]
lgtm :+1:
